[Feat/TK-177] 정산 취소 API 연동 및 지출 세부 내역 퍼블리싱 수정

yaejinkong opened the pull request:
## 📋 연관된 이슈 번호

- close #66 

## 🛠 구현 사항

- 정산 취소 API 연동
- 정산 취소 완료 및 동의 완료 페이지 퍼블리싱 

## 📚 변경 사항

- 지출 세부 내역 퍼블리싱 수정

## 🏜 스크린샷

- 지출 세부 내역 페이지 
<img width="354" alt="image" src="https://github.com/user-attachments/assets/1caae438-b529-4e85-b850-c9f5705bb8fd">

- 상세 내역 열기 시 지출 세부 내역 페이지
<img width="354" alt="image" src="https://github.com/user-attachments/assets/3ad1f331-51cc-4176-b877-7386f6aed221">

- 정산 동의 완료 시 지출 세부 내역 페이지
<img width="353" alt="image" src="https://github.com/user-attachments/assets/5d9de9dc-255f-431a-86bd-7c507f74cf87">

- 정산 취소 완료시 페이지
<img width="355" alt="image" src="https://github.com/user-attachments/assets/0012243c-72a1-4c07-8d71-30bb68c623c8">

## 💬 To Reviewers
- 정산 보류 페이지는 정산 동의 API 연동 pr에도 추가해둬서 그 쪽에서 봐주시면 될 거 같습니다!
- 피드백 부탁드려요!

yaejinkong committed:
[refactor/TK-177]: 알림창 css 수정

yaejinkong committed:
[feat/TK-177]: 정산 취소 API 연동

yaejinkong committed:
[feat/TK-177]: 정산 취소 완료 페이지 완성

yaejinkong committed:
[feat/TK-177]: 정산 보류 페이지 완성

yaejinkong committed:
[refactor/TK-177]: 정산 동의 완료 시 여행 홈으로 돌아가도록 변경

yaejinkong committed:
Merge remote-tracking branch 'origin/dev' into feat/TK-177-settlement-cancel

github-actions[bot] committed:
chore: apply Prettier formatting

yaejinkong committed:
[refactor/TK-177]: @withbee/exception 사용하도록 주석 제거

yaejinkong committed:
Merge remote-tracking branch 'origin/feat/TK-177-settlement-cancel' into feat/TK-177-settlement-cancel

doyi0107 commented on apps/mobile/components/ModalWrapper.tsx:
```diff
@@ -4,9 +4,40 @@ import { Button } from '@withbee/ui/button';
+  const router = useRouter();
+
+  const handelCancelSettlement = async () => {
+    try {
```
try-catch문은 instance에 있어서 생략 가능합니다!

yaejinkong replied:
넵 수정하겠습니다!

doyi0107 commented on apps/mobile/components/ModalWrapper.tsx:
```diff
@@ -4,9 +4,40 @@ import { Button } from '@withbee/ui/button';
+      showToast.success({
+        message: '정산이 취소되었습니다.',
+      });
+      console.log(response);
```
확인한 log들은 삭제하는 게 좋을 것 같아요!

yaejinkong replied:
넵 삭제하겠습니당!!

yaejinkong committed:
[refactor/TK-177]: try, catch 문 삭제

- 인스턴스 내부에 try, catch 문이 있어서 삭제하도록 변경
- 확인한 로그 삭제

yaejinkong committed:
Merge remote-tracking branch 'origin/dev' into feat/TK-177-settlement-cancel

yaejinkong committed:
[refactor/TK-177]: cache: 'no-store' 다시 추가

yaejinkong committed:
[refactor/TK-177]: resolved merge conflicts

yaejinkong committed:
[refactor/TK-177]: 페이지 내에서 이동되도록 target="_blank" 삭제

yaejinkong committed:
[refactor/TK-177]: 세부 지출 내역 디테일 수정

- 최종 결제 금액에 "총" 붙이기
- 필드 명칭 변경
- 간격 조정
- 구구절절 코멘트 삭제
- 빨간색 색상 변경
- 정산 완료 시 버튼 비활성화
- 상호명이 14자 이상시 ...으로 표시

yaejinkong committed:
[refactor/TK-177]: 세부 지출 내역 디테일 수정

- width를 45% -> 50%로 변경

yaejinkong committed:
[refactor/TK-177]: settlementService.ts에 agreed 필드 타입 추가

yaejinkong committed:
[refactor/TK-177]: 유효하지 않은 정산 요청 에러 처리

yaejinkong committed:
[refactor/TK-177]: 정산 완료 버튼 fixed로 변경

yaejinkong committed:
[refactor/TK-177]: 지출 상세 내역 시간 형식 변경

- 구분자를 -로 변경
- 초 없앰
- 상호명 제한길이 14자 -> 13자로 변경

yaejinkong committed:
Merge remote-tracking branch 'origin/dev' into feat/TK-177-settlement-cancel

yaejinkong committed:
[refactor/TK-177]: 존재하지 않은 여행id로 정산 요청할 경우 에러 처리

yaejinkong committed:
[refactor/TK-177]: 정산 요청 오류 시 돌아가기 버튼의 라우팅 주소 변경
